library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3311,6 +3311,13 @@
     <string name="device_manager_other_sessions_no_unverified_sessions_found">No unverified sessions found.</string>
     <string name="device_manager_other_sessions_no_inactive_sessions_found">No inactive sessions found.</string>
     <string name="device_manager_other_sessions_clear_filter">Clear Filter</string>
+    <string name="device_manager_other_sessions_select">Select sessions</string>
+    <string name="device_manager_other_sessions_select_all">Select all</string>
+    <string name="device_manager_other_sessions_deselect_all">Deselect all</string>
+    <plurals name="device_manager_other_sessions_selected">
+        <item quantity="one">%1$d selected</item>
+        <item quantity="other">%1$d selected</item>
+    </plurals>
```

Comment:
Same remark about generic key. Maybe rename to `x_selected`

Reply:
Updated here: [95de903](https://github.com/vector-im/element-android/pull/7408/commits/95de903af022f5e837c7540594102d7c1cbca7c8)